bugsnag-plugin-android-ndk/src/main/jni/event.h

```diff
@@ -178,6 +185,12 @@ typedef struct {
   char url[64];
 } bsg_notifier;
 
+typedef struct {
+  pid_t id;
+  char name[16];
```

Comment:
Will this definitely be large enough to fit any thread name?

Reply:
This mirrors the Kernel process name, as it'll report in `/proc/stat`. Happy to increase it, but it might be a bit of waste?